lib/autokey/service.py

```diff
@@ -507,7 +507,7 @@ def execute_path(self, path: pathlib.Path, buffer=''):
         # Overwrite __file__ to contain the path to the user script instead of the path to this service.py file.
         scope["__file__"] = path.resolve()
         try:
-            exec(open(path).read(), scope)
+            exec(pathlib.Path(path).read_text(), scope)
```

Comment:
Isn’t `path` already a Path instance? You call `.resolve()` in line 508, so it must be. So `exec(path.read_text(), scope)` should be sufficient here.

Reply:
Oops, thanks

Reply:
Fixed